apps/web/test/api-comments-validation.test.ts

```
@@ -0,0 +1,611 @@
+		}
+	}
+
+	return { valid: true }
+}
+
+describe('Comments API Validation Logic', () => {
+	beforeEach(() => {
+		// Reset all mocks
+		Object.values(mockSupabase).forEach((fn) => {
+			// bun:test mock has a `mock` property with `clear`
+			if (typeof (fn as any)?.mock?.clear === 'function') {
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)